browser_history/__init__.py

```diff
@@ -1,6 +1,11 @@
 import inspect
+import webbrowser
+
 from . import browsers, generic, utils  # noqa: F401
 
+if utils.get_platform() == utils.Platform.WINDOWS:
+    from winreg import HKEY_CURRENT_USER, OpenKey, QueryValueEx  # type: ignore
```

Comment:
I didn't notice this before, but here the namespace is getting polluted by these windows imports, as well as `webbrowser` and `inspect`.

![image](https://user-images.githubusercontent.com/34161949/104892867-b1557180-5998-11eb-819a-3e06f21a6371.png)

Reply:
I moved all the functions to `utils` and, IMO, it looks better this way. I did not run into any circular import errors either.

![image](https://user-images.githubusercontent.com/34161949/104894130-4dcc4380-599a-11eb-9f40-b20a65349575.png)

Reply:
Yeah, I don't know where the circular import issue was coming from before. I have moved them to `utils.py` and everything was okay :+1: